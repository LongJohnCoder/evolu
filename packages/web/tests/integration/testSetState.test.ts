import { pageDom, pageGoto } from './helpers';

beforeEach(async () => {
  await pageGoto('testSetState');
});

test('select', async () => {
  await expect(await pageDom()).toMatchSnapshot();
});
