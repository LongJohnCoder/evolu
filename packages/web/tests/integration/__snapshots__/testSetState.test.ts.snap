// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`select 1`] = `
<div
  id="__next"
>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    tabindex="0"
  >
    <div
      class="root"
    >
      <div
        class="paragraph"
      >
        ·foo·
      </div>
    </div>
  </div>
  <pre
    class="jsx-2255726786"
  >
    <span
      class="jsx-1782790912"
      title="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'paragraph'},'children':[{'text':'foo'}]}]}"
    >
      ·2·
      · ·
      ·{"hasFocus":true,"selection":{"anchor":[0,0,1],"focus":[0,0,3]}}·
    </span>
    <span
      class="jsx-1782790912"
      title="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'paragraph'},'children':[{'text':'paragraph'}]}]}"
    >
      ·1·
      · ·
      ·{"hasFocus":true,"selection":{"anchor":[0,0,0],"focus":[0,0,0]}}·
    </span>
    <span
      class="jsx-1782790912"
      title="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'paragraph'},'children':[{'text':'paragraph'}]}]}"
    >
      ·0·
      · ·
      ·{"hasFocus":true}·
    </span>
  </pre>
</div>
`;
